jira_agile_metrics/config_test.py

```diff
@@ -482,16 +476,15 @@ def test_config_to_options_extends():
 
         # overridden
         assert options['connection']['domain'] == 'https://bar.com'
-        
+
         # from extended base
         assert options['settings']['backlog_column'] == 'Backlog'
-        assert options['settings']['committed_column'] == 'Committed'
-        assert options['settings']['final_column'] == 'Test'
+        assert options['settings']['committed_column'] == 'In progress'
```

Comment:
Again - changes a test that would have passed previously. This one makes me more nervous because it looks like it's genuinely picking a different column as the "Committed" one without changing the test setup, which surely means a regression for existing configurations?

Reply:
See https://github.com/DeloitteDigitalUK/jira-agile-metrics/pull/27#discussion_r432839745.